### ISSUE TITLE
afterSlideLoad on first slide

### ISSUE THREAD

**Issue by ascottmccauley** [alvarotrigo/fullPage.js#1309]
For consistency, I think `afterSlideLoad` should always be called when a slide gets loaded. right now it only gets called when changing horizontal slides, and not on the first slide.

Please add this minor enhancement.

**Comment by ascottmccauley** [comment]
'First Slide' vs. 'Page Load'

I think that the afterSlideLoad should also get called when changing `sections` that have `slides`. 
In other words 'every time a slide gets loaded.'

**Comment by jameslanman** [comment]
Yes - this would be super useful if one wanted to apply the same effect to every new slide.

**Comment by deckikwok** [comment]
totally agreed. such a weird behavior.

**Comment by cumuluswebdesign** [comment]
This would offer a lot of flexibilty, it's the missing feature I'm looking for!

**Comment by alvarotrigo** [comment]
For now, you can use:

```javascript
afterLoad: function(anchorLink, index){
    //loading first slide of a section?
    if($(this).find('.slide.active').index() === 0){
         console.log("first slide has been loaded");
    }
},
```

**Comment by cumuluswebdesign** [comment]
Thanks for the fast response. To simply get the next slide index I did this:

```javascript
onLeave: function(index, nextIndex, direction)
{
      // Get next slide index
      var slideIndex = $('#section' + nextIndex).find('.slide.active').index() ;
      // Do something with this slideIndex ...
}
```

**Comment by K1llMan** [comment]
Any chance to have 'afterSlideLoad' event on section change?

**Comment by alvarotrigo** [comment]
> Any chance to have 'afterSlideLoad' event on section change?

Not at the moment. But see my [response above](https://github.com/alvarotrigo/fullPage.js/issues/1309#issuecomment-186431865) for a workaround.
